earth_enterprise/src/fusion/autoingest/sysman/khAssetManager.cpp

```diff
@@ -1236,11 +1233,6 @@ std::string khAssetManager::RetrieveTasking(const FusionConnection::RecvPacket&
 	      .arg(ToQString(GET_TASKS)));
 	  }
   }
-  catch (khTimedMutexException e) {
-    // Replying with a string beginning "ERROR:" passes an exception message back to the caller
-    // alternatively we could throw an exception but that could flood fusion logs with warnings
-    replyPayload = sysManBusyMsg;
```

Comment:
Minor: probably should get rid of `sysManBusyMsg`. I doubt it's used anywhere else.